spec/models/saving_goal_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe SavingGoal, type: :model do
+  it { should have_db_column :amount}
+
+  describe 'Factory' do
+    it 'should have valid Factory' do
+      expect(FactoryBot.create(:income)).to be_valid
```

Comment:
why dont you test that the saving_goal factory is valid here?

Reply:
As @oliverochman mentions in his previous comment... Why are we creating an `income` object in this test instead of `saving_goal`? This line here is testing the wrong thing.